src/resources/drives.jsx

```diff
@@ -0,0 +1,64 @@
+import { 
+    List, 
+    ReferenceField,
+    TextField, 
+    Datagrid,
+    Edit,
+    SimpleForm,
+    DateField,
+    ReferenceInput,
+    TextInput,
+    BooleanField,
+    BooleanInput,
+    NumberInput,
+    Create,
+    DateTimeInput
+} from "react-admin";
+
+const driveFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+export const DriveList = () => (
+  <List filters={driveFilters}>
+    <Datagrid rowClick="edit">
+      <TextField source="id" />
+      <ReferenceField source="driver_id" reference="profiles" />
+      <DateField source="start_time" showTime />
+      <TextField source="start" />
+      <DateField source="end_time" showTime />
+      <TextField source="end" />
+      <BooleanField source="cancelled" />
+    </Datagrid>
+  </List>
+);
+
+export const DriveEdit = () => (
```

Comment:
By the way, the title can be fixed: https://marmelab.com/react-admin/Edit.html#title

Reply:
thanks did that